indico/htdocs/js/indico/jquery/itempicker.js

```diff
@@ -0,0 +1,221 @@
+                            } else {
+                                self._handleSelect($item, itemData);
+                            }
+                        }
+                    }
+                });
+
+                if (itemData.colors) {
+                    itemIcon.css('color', '#' + itemData.colors.background);
+                }
+
+                $item.append($('<span>', {'class': self.options.checkedItemIcon + ' checked-item-icon'}));
+                $item.append(itemIcon).append($('<span>', {'class': 'item-title', 'text': itemData.title}));
+
+                if (isSelected) {
+                    self._markItemAsSelected($item, itemData);
+                    itemsContainer.prepend($item);
+                } else {
+                    itemsContainer.append($item);
+                }
+            });
+
+            dropdownContainer.append(itemsContainer);
+            if (self.options.footerElements) {
+                self._appendFooterItems(dropdownContainer);
+            }
+
+            return dropdownContainer;
+        },
+
+        refreshItemList: function(items) {
+            this.element.qtip('option', 'content.text', this._createQtipContent(items));
+        },
+
+        hide: function() {
+            this.element.qtip('hide');
+        },
+
+        _handleSelect: function(item, itemData) {
+            this.element.data('selected-item-id', itemData.id);
+            this._markItemAsSelected(item, itemData);
+            if ($.isFunction(this.options.onSelect)) {
+                this.options.onSelect.call(this.element, itemData);
+            }
+        },
+
+        _markItemAsSelected: function(item, itemData) {
+            var bgColor = itemData.colors ? itemData.colors.background : this.options.selectedItemBackground;
+            var itemsContainer = item.closest('.dropdown-items-container');
+            itemsContainer.find('.dropdown-item').css('background', '');
+            itemsContainer.find('.checked-item-icon.active').removeClass('active');
+            item.find('.checked-item-icon').addClass('active');
+            item.css('background', this._increaseBrightness(bgColor, 50));
+        },
+
+        _deselectItem: function(item, itemData) {
+            item.css('background', '').find('.checked-item-icon').removeClass('active');
+        },
+
+        _increaseBrightness(hex, percent) {
+            // from http://stackoverflow.com/questions/6443990/javascript-calculate-brighter-colour
+            // strip the leading # if it's there
+            hex = hex.replace(/^\s*#|\s*$/g, '');
+
+            // convert 3 char codes --> 6, e.g. `E0F` --> `EE00FF`
+            if (hex.length == 3) {
+                hex = hex.replace(/(.)/g, '$1$1');
+            }
+
+            var r = parseInt(hex.substr(0, 2), 16),
+                g = parseInt(hex.substr(2, 2), 16),
+                b = parseInt(hex.substr(4, 2), 16);
+
+            return '#' +
```

Comment:
I would do that outside for consistency with `itemIcon.css('color', '#' + itemData.colors.background)`